docs/source/concepts/scripting_workspaces.rst

```diff
@@ -0,0 +1,120 @@
+.. _scripting_workspaces:
+
+=====================
+Workspaces in Scripts
+=====================
+
+
+The :ref:`Workspaces Toolbox<WorkbenchWorkspaceToolbox>` is used to store and manage workspaces within Mantid. Behind the scences, the Analysis Data Service or ADS is used to control these workspaces. 
+
+Below are some examples of how to control workspaces with a script.
+
+.. contents:: Table of contents
+    :local:
+
+
+
+Relevant Algorithms
+===================
+
+Required import:
+
+.. code-block:: python
+
+    from mantid.simpleapi import *
+
+:ref:`Load a Workspace <algm-Load>` from a file:
+
+.. code-block:: python
+
+    ws = Load('EMU00020884.nxs')
+
+:ref:`Create a Workspace <algm-CreateWorkspace>`:
+
+.. code-block:: python
+
+    ws = CreateWorkspace(DataX=X, DataY=Y, DataE=E, NSpec=4, UnitX="Wavelength")
+
+:ref:`Delete a Workspace <algm-DeleteWorkspace>`:
+
+.. code-block:: python
+
+    DeleteWorkspace(ws)
+
+:ref:`Group Workspaces <WorkspaceGroup>`:
+
+.. code-block:: python
+
+    wsGroup = GroupWorkspaces("ws1,ws2,ws3")
```

Comment:
Probably better to encourage people to use lists rather than long strings
```suggestion
    ws1 = CreateSampleWorkspace()
    ws2 = CreateSampleWorkspace()
    ws3 = CreateSampleWorkspace()
    wsGroup = GroupWorkspaces([ws1,ws2,ws3])
    
    # or if you only have the names of the workspaces 
    wsGroup = GroupWorkspaces(['ws1','ws2','ws3'])
```